lnprototest/backend/bitcoind.py

```diff
@@ -144,11 +159,11 @@ def start(self) -> None:
         self.rpc.generatetoaddress(100, self.rpc.getnewaddress())
 
     def stop(self) -> None:
+        self.rpc.stop()
         self.proc.kill()
         shutil.rmtree(os.path.join(self.bitcoin_dir, "regtest"))
+        self.rpc = None
```

Comment:
You should remove the test for self.rpc is None in start().  Since it's always None now, that's just confusing.

(Setting self.rpc to None is probably just confusing now...)